### PR TITLE
Changes to move from private preview to Beta

### PR DESCRIPTION
**Pull request by yvsubhash**
Changes to move this release from Private Preview to Beta.

Preview: [https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/addons-api-reference?branch=pr-en-us-1790](https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/addons-api-reference?branch=pr-en-us-1790)

[https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-1790](https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-1790)

### CODE REVIEW

File: microsoft-edge/extensions-chromium/publish/api/using-addons-api.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ---
-title: Using the Microsoft Edge Add-ons API (in private preview)
+title: Using the Microsoft Edge Add-ons API (is currently in bea)
```
Comment (ashishnirvana1) [inline microsoft-edge/extensions-chromium/publish/api/using-addons-api.md:2]: The spelling of beta is incorrect @yvsubhash

**Review by vchapel — APPROVED** [review]
LGTM

File: microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
 ---
-title: Microsoft Edge Add-ons API Reference (in private preview)
+title: Microsoft Edge Add-ons API Reference (is currently in beta)
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md:2]: please change "title:" yaml field x2, titles x2, link text in all .md files that point to these articles, and the TOC titles x2, from (is currently in beta) to simply (Beta)

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Not ready to merge, TOC still says (Preview) instead of (Beta)

**Review by vchapel — APPROVED** [review]
LGTM

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**